lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,437 @@
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.subject
+
+  - name: context
+    required: 'Yes'
+    type: Reference
+    description: The encounter associated with the charge.
+    action: modify
+    example: |
+      {
+        "context": {
+          "reference": "Encounter/97965088"
+        }
+      }
+    note: >
+      <ul>
+        <li>Must be an Encounter reference</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.context
+
+  - name: occurrenceDateTime
+    required: 'No'
+    type: dateTime
+    description: Date/time when the charged service was applied.
+    action: modify
+    example: |
+      {
+        "occurrenceDateTime": "2021-01-08T10:38:00Z"
+      }
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.occurrence_x_
+
+  - name: quantity
+    required: 'No'
+    type: Quantity
+    description: Quantity of which the charge item has been serviced.
+    action: modify
+    example: |
+      {
+        "quantity": {
+          "value": 31
+        }
+      }
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.quantity
+
+  - name: code
+    required: 'Yes'
+    type: CodeableConcept
+    description: The primary billing code that identifies the charge.
+    action: modify
+    example: |
+      {
+        "code": {
+          "coding": [
+            {
+              "extension": [
+                {
+                  "valueCoding": {
+                    "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
+                    "code": "667687"
+                  },
+                  "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                }
+              ],
+              "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-CDM_SCHED",
+              "code": "64897"
+            }
+          ]
+        }
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the CDM_SCHED, CPT, HCPCS, or PROCCODE proprietary Bill Codes Types systems.</li>
+        <li>The Bill Code Type provided in `code` must be the highest tier of all of the Bill Code Types associated to the ChargeItem</li>
+        <li>Tiering logic from highest to lowest: CDM_SCHED, CPT, HCPCS, or PROCCODE</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.code
+
+  - name: performer
+    required: 'No'
+    type: List of BackboneElement
+    description: Indicates who performed or participated in the charge.
+    action: modify
+    example: |
+      {
+        "performer": [
+          {
+            "function": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/v2-0912",
+                  "code": "OP",
+                  "display": "Ordering Provider"
```

Comment:
Do we actually support/validate the display and the text for the performer function on a create, or are we just looking at the code value and how it is mapped to system? What would happen if I provided code= OP w/out display and text? What would happen if I provided code= OP w/ just "Provider" in the display and text fields?

Reply:
I am not sure about create but I just verified modify request and when we try to not pass display and text it is going to local service as we are not validating them. We are just validating its type and type should be string for both display and text fields. for your second question if we pass "Provider" in both display and text it should pass on our side as we only validate whether they are string or not. 
Note: I tested it in dev and it is giving 500 error from local service but i assume it is different issue I guess.
@exopsRamba can you please provide your thoughts as well for making sure what I mentioned above is accurate or not.

Reply:
It is my understanding that we don't generally validate the display and text fields are valid string values in other resources as well. 
I looked into the x-request-id for Anurag's testing on this thread (8e4e41fa-6a44-467d-b09c-a0fb89c02e87) and the request seems to error out on an 'invalid identifier' somewhere on the localservice side.

Reply:
after trying it on 64DEV without providing text and display for performer it still got 201 response with performer in the newly created charge.